common-util/AbiAndAddresses/SignMessageLib.jsx

```diff
@@ -0,0 +1,52 @@
+export const SIGN_MESSAGE_LIB_CONTRACT = {
```

Comment:
This is not needed at this point of time, but let it be just in case of future needs.